gradle.properties

```diff
@@ -174,3 +174,5 @@ xercesImpl.version=2.9.1
 systemProp.sonar.organization=apache
 systemProp.sonar.projectKey=apache_calcite
 systemProp.sonar.host.url=https://sonarcloud.io
+
+kotlin.jvm.target.validation.mode = IGNORE
```

Comment:
This is a result of a weird Kotlin bug https://youtrack.jetbrains.com/issue/KT-48745
which is unlikely to be fixed

Reply:
finally was able to make it working without this property